xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/notification/rss/default.vm

```diff
@@ -0,0 +1,51 @@
+### Default plain text template for RSS feeds
+#template('notification/macros.vm')
+###
+### Get the app
+###
+#set ($app  = "$!event.events.get(0).application")
+#if ($app == 'xwiki')
+  #set ($app = $services.localization.render('core.events.appName'))
+#end
+###
+### Get the doc
+###
+#set ($document = $xwiki.getDocument($event.document))
+###
+### Display
+###
+<p>
+  <b>$app: [$event.type]</b>
```

Comment:
Minor: On RSS, what is the HTML version we need to support?

If it's XHTML, I would use <strong> instead of <b>.